protocols/enocean/org.eclipse.smarthome.protocols.enocean.basedriver.impl/about.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
```

Comment:
This about.html is not packaged within the binary jar by the build, but it should be.